packages/documentation/pages/examples/layouts.vue

```diff
@@ -22,7 +23,22 @@
 			headerTitle="LayoutContainer Example"
 			:menu="actionbarMenu"
 		/>
-		<div slot="workspace"></div>
+		<div slot="workspace">
+			<KtForm>
```

Comment:
you can choose to use a field without a form if the formKey is NONE anyways

so no need for `KtForm` and thus `formKey`

Reply:
So no `formKey` on field and no `Form` ? Cool!